### PR TITLE
Resize for popup for consistent size on Windows and Mac OS

### PR DESCRIPTION
**Pull request by F-OBrien**
Due to the differences in the way that Windows and Mac OS renders the window frames and borders it results in inconsistent popup sizes which obscures some information. e.g.  the "Ask again later" option as shown below for Windows. This only happens when a popup is created and not when clicking on the extension icon. `chrome.windows.create` doesn't have a clean way to handle both OS's. This PR is a hacky way to try to handle both OS's.

Original `chrome.windows.create` outer dimensions = 560x621 (from `POPUP_WINDOW_OPTS`)
Resulting Windows popup inner dimension = 544x582
image

After resize to 560x600 inner dimension
image

### CODE REVIEW

**Comment by Tbaut** [comment]
I tested it on Linux and the hack doesn't seem to work. Here's a QR code request: 
image

**Comment by jacogr** [comment]
... These window sizes between browsers and platforms has been a huge PITA since the start...

**Comment by F-OBrien** [comment]
That's disappointing. Well it seems to work with Chrome at least if you wanted to consider it as a slight improvement over the current implementation.

**Comment by jacogr** [comment]
I'll defer to @Tbaut on this one since (like he mentioned above), he spent a lot of time on this in the past.

My general feeling is that - if it is better in some cases and no worse in others (e.g. FF), it is certainly worth a merge. 

TL;DR As long as it doesn't make anything worse, I'm ok with it.
